client/gutenberg/extensions/related-posts/edit.jsx

```diff
@@ -17,95 +17,94 @@ export const MAX_POSTS_TO_SHOW = 6;
 
 function PlaceholderPostEdit( props ) {
 	return (
-		<div
+		<ul
 			className="jp-related-posts-i2__post"
 			id={ props.id }
 			aria-labelledby={ props.id + '-heading' }
 		>
-			<strong id={ props.id + '-heading' } className="jp-related-posts-i2__post-link">
-				{ __( 'Preview: Not enough related posts found' ) }
-			</strong>
+			<li className="jp-related-posts-i2__post-link">
+				<a id={ props.id + '-heading' } href={ window.location }>
+					{ __( 'Preview: Not enough related posts found' ) }
+				</a>
+			</li>
 			{ props.displayThumbnails && (
-				<figure
-					className="jp-related-posts-i2__post-image-placeholder"
-					aria-label={ __( 'Placeholder image' ) }
-				>
-					<SVG
-						className="jp-related-posts-i2__post-image-placeholder-square"
-						xmlns="http://www.w3.org/2000/svg"
-						width="100%"
-						height="100%"
-						viewBox="0 0 350 200"
+				<li className="jp-related-posts-i2__post-image-placeholder">
+					<figure
+						className="jp-related-posts-i2__post-image-placeholder-figure"
+						aria-label={ __( 'Placeholder image' ) }
 					>
-						<title>{ __( 'Grey square' ) }</title>
-						<Path d="M0 0h350v200H0z" fill="#8B8B96" fill-opacity=".1" />
-					</SVG>
-					<SVG
-						className="jp-related-posts-i2__post-image-placeholder-icon"
-						xmlns="http://www.w3.org/2000/svg"
-						width="24"
-						height="24"
-						viewBox="0 0 24 24"
-					>
-						<title>{ __( 'Icon for image' ) }</title>
-						<Path fill="none" d="M0 0h24v24H0V0z" />
-						<Path d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm-4.86 8.86l-3 3.87L9 13.14 6 17h12l-3.86-5.14z" />
-					</SVG>
-				</figure>
+						<SVG
+							className="jp-related-posts-i2__post-image-placeholder-figure-square"
+							xmlns="http://www.w3.org/2000/svg"
+							width="100%"
+							height="100%"
+							viewBox="0 0 350 200"
+						>
+							<title>{ __( 'Grey square' ) }</title>
+							<Path d="M0 0h350v200H0z" fill="#8B8B96" fill-opacity=".1" />
+						</SVG>
+						<SVG
+							className="jp-related-posts-i2__post-image-placeholder-figure-icon"
+							xmlns="http://www.w3.org/2000/svg"
+							width="24"
+							height="24"
+							viewBox="0 0 24 24"
+						>
+							<title>{ __( 'Icon for image' ) }</title>
```

Comment:
I know this wasn't introduced in this PR, but noting that `title` is basically ignored by screen readers because `SVG` component hides the whole icon from screen readers.
```
'aria-hidden': 'true',
focusable: 'false',
```
https://github.com/WordPress/gutenberg/blob/e9aac1490cf02ce747eaa6c7291855dba14a1663/packages/components/src/primitives/svg/index.js#L18-L20

Reply:
This is why there is an `aria-label` in the `figure` :)

Reply:
Sure but isn't `<title>` still redundant? 🙃

Reply:
Also, the `Svg` component in Gutenberg takes care of the `aria-hidden` and `focusable` stuff automatically.

Reply:
Yes, so that's what I meant to say that `<title>` isn't needed here at all, because `SVG` component adds `aria-hidden` automatically and thus screen readers will never actually read that `title`. Unless I'm missing something? :-)

Reply:
True, but it annoys the linter.

Reply:
I'm confused, which linter/rule? 🙃